backend/src/main/kotlin/org/loculus/backend/model/ReleasedDataModel.kt

```diff
@@ -93,6 +109,8 @@ open class ReleasedDataModel(
             NullNode.getInstance()
         }
 
+        val earliestReleaseDate = earliestReleaseDateFinder?.calculateEarliestReleaseDate(rawProcessedData)
```

Comment:
It would be more in line with how we compute other same-accession across-version fields to make a db query and follow the pattern for `getLatestVersions` and `getLatestRevocationVersions` as part of `computeVersionStatus` and `computeDataUseTerm` above

Relying on accession/versions to be sorted is a bit brittle and probably quite a few more lines of code as well.

Reply:
the examples you gave don't access JSON fields, so it's not quite the same. Personally I'm not sure whether it's possible to do this with SQL -- now I think it might, but it would definitely be a very complicated query.

I think the way it works now doesn't introduce any problems?